bundles/org.openhab.binding.doorbird/src/main/java/org/openhab/binding/doorbird/internal/DoorbirdConfiguration.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.doorbird.internal;
+
+/**
+ * The {@link DoorbirdConfig} class contains fields mapping thing configuration parameters.
+ *
+ * @author Mark Hilbush - Initial contribution
+ */
+public class DoorbirdConfiguration {
+    /**
+     * Doorbird ID must match the ID returned in UDP events
+     */
+    public String doorbirdId;
+
+    /**
+     * Hostname or IP address of doorbell
+     */
+    public String doorbirdHost;
+
+    /**
+     * User ID used for API requests
+     */
+    public String userId;
+
+    /**
+     * Password used in API requests, and to decrypt doorbird events
+     */
+    public String userPassword;
+
+    /**
+     * Rate at which image channel will be updated
+     */
+    public Integer imageRefreshRate;
```

Comment:
Since this parameter (and a few other ones) are optional, I am testing for the value of null to know whether or not the config parameter is set. Is there another way that's more preferred?

Reply:
I would set a default to 0 and check `>0` instead of `==null`.

Reply:
I understand your perspective, and that will certainly work. But, from a UI perspective, doesn't it make more sense for the field to be blank, rather than 0? A blank field would tend to suggest the feature (e.g. periodic image refresh) is unused. It just feels a bit more intuitive to me. WDYT?

Reply:
Empty will work also. We already have (at least) nikohomecontrol, sonyaudio, onkyo, dmx, modbus, velbus, neeo where "refresh=0" means "disable refresh".